Source/Scene/Cesium3DTileStyle.js

```diff
@@ -344,6 +421,270 @@ define([
         },
 
         /**
+         * Gets or sets the {@link StyleExpression} object used to evaluate the style's <code>outlineColor</code> property.
+         * <p>
+         * The expression must return a <code>Color</code>.
+         * </p>
+         *
+         * @memberof Cesium3DTileStyle.prototype
+         *
+         * @type {StyleExpression}
+         *
+         * @exception {DeveloperError} The style is not loaded.  Use Cesium3DTileStyle.readyPromise or wait for Cesium3DTileStyle.ready to be true.
+         *
+         * @example
+         * var style = new Cesium3DTileStyle({
+         *     outlineColor : '(${Temperature} > 90) ? color('red') : color('white')'
+         * });
+         * style.outlineColor.evaluate(frameState, feature); // returns a Cesium.Color
+         *
+         * @example
+         * var style = new Cesium.Cesium3DTileStyle();
+         * // Override outlineColor expression with a custom function
+         * style.outlineColor = {
+         *     evaluate : function(frameState, feature) {
+         *         return Color.WHITE;
+         *     }
+         * };
+         *
+         * @see {@link https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling|3D Tiles Styling language}
+         */
+        outlineColor : {
```

Comment:
Can you run coverage for these get/set functions?  Most of them are not covered.